docs/example.ts

```diff
@@ -91,28 +76,30 @@ const mnemonicForAttester = Kilt.Identity.generateMnemonic()
 const attester = Kilt.Identity.buildFromMnemonic(mnemonicForAttester)
 
 // And we need to build a request for an attestation
-const requestForAttestation = new Kilt.RequestForAttestation(claim, [], claimer)
+const requestForAttestation = Kilt.RequestForAttestation.fromClaimAndIdentity(
+  claim,
+  claimer
+)
 
 // Excourse to the messaging system
```

Comment:
oh and excourse : not a word

Reply:
I'm not sure what they wanted to say. I changed it to `Excursion to the messaging system`.